cms/static/js/spec/views/settings/main_spec.js

```diff
@@ -186,5 +186,41 @@ define([
             AjaxHelpers.expectJsonRequest(requests, 'POST', urlRoot, modelData);
         });
 
+        describe("enrollmentEndEditable set to true", function() {
```

Comment:
OK, I know I said nit, but it really makes more sense to me to have both of these test cases nested within a single describe. The general pattern is to have the describe be about a feature, and then the test cases related to the permutations be within that. The feature here is that editability of the course enrollment end date is controlled by the enrollmentEndEditable flag.